source/css/style.css.scss

```diff
@@ -8,3 +8,45 @@ a {
 a:hover {
   color: $primary-dark;
 }
+
+h1 {
+  color: $primary-color
+}
+
+.container {
+  text-align: justify;
+  text-justify: inter-word;
+  display: flex;
+}
+
+div.left {
```

Comment:
Po to masz kolumny w bootstrapie
```haml
.row
  .col-md-4
    Lewo
  .col-md-4
    Środek
  .col-md-4
    Prawo
```

albo nawet klasy `float-left` i `float-right`